e2e/tests/notebook/notebook-create-python.spec.ts

```diff
@@ -58,21 +58,27 @@ describe('Create python kernel notebook', () => {
       codeFile: 'resources/python-code/import-os.py'
     });
     // toContain() is not a strong enough check: error text also includes "success" because it's in the code
-    expect(cell1OutputText).toMatch(/success$/);
-
-    expect(await notebook.runCodeCell(2, {
-      codeFile: 'resources/python-code/import-libs.py'
-    })).toMatch(/success$/);
-
-    expect(await notebook.runCodeCell(3, {
-      codeFile: 'resources/python-code/git-ignore-check.py',
-      markdownWorkaround: true
-    })).toMatch(/success$/);
-
-    expect(await notebook.runCodeCell(4, {
-      codeFile: 'resources/python-code/nbstripoutput-filter.py',
-      markdownWorkaround: true
-    })).toMatch(/success$/);
+    await expect(cell1OutputText).toMatch(/success$/);
```

Comment:
It was a warning: `Missing await for an async function call`.

<img width="499" alt="Screen Shot 2021-08-09 at 3 14 54 PM" src="https://user-images.githubusercontent.com/35533885/128761591-22101ac4-bc18-4a94-9124-55ed62c62a64.png">

Reply:
🤷  I don't get it, but I don't feel strongly enough to dig deeper on this

Reply:
Because `expect-puppeteer` assertion library is packaged in `jest-puppeteer`.  And it provides a `toMatch` method which is a [async func](https://github.com/smooth-code/jest-puppeteer/blob/a4d6a2537ffc88bf5ad9a87e880474fbe0ee4a05/packages/expect-puppeteer/src/matchers/toMatch.js#L4).

It's not the jest `toMatch` method used in test ( [non-async](https://jestjs.io/docs/expect#tomatchregexp--string) ).

Reply:
Ahhh, makes sense. Thank you for clarifying